elementary/clients/gcs/client.py

```diff
@@ -23,8 +23,10 @@ def create_client(cls, config: Config) -> Optional['GCSClient']:
     def send_report(self, local_html_file_path: str, remote_bucket_file_path: Optional[str] = None) -> bool:
         report_filename = bucket_path.basename(remote_bucket_file_path) if remote_bucket_file_path else path.basename(local_html_file_path)
         bucket_report_path = remote_bucket_file_path if remote_bucket_file_path else report_filename
+        logger.info("Pushing report file %s to bucket %s.", bucket_report_path, self.config.gcs_bucket_name)
```

Comment:
bucket_report_path could contain a custom path in the target bucket so I suggest to us the report_filename as the file name in this log

Reply:
Made the change:
```python
        logger.info("Pushing report file %s to bucket %s.", report_filename, self.config.gcs_bucket_name)

```